src/main/java/transact/model/transaction/Transaction.java

```diff
@@ -77,11 +111,11 @@ public TransactionId getTransactionId() {
     }
 
     public Boolean hasPersonInfo() {
-        return this.person != null;
+        return !person.equals(Person.NullPerson);
```

Comment:
```suggestion
        return person != Person.NullPerson;
```

Will throw NPE if person is NullPerson since we have this in Person.equals method:
```java
return name.equals(otherPerson.name)
                && phone.equals(otherPerson.phone)
                && email.equals(otherPerson.email)
                && address.equals(otherPerson.address)
                && tags.equals(otherPerson.tags);
```
We will have only 1 instance of NullPerson anyways so reference equality should be ok?